presto-cache/pom.xml

```diff
@@ -133,4 +133,24 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
```

Comment:
This seems unrelated? Do we need this?

Reply:
explained above. for some reason on snapshot alluxio, enforcer will fail on build